libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -469,6 +477,8 @@ void AP_ADSB::set_vehicle(const uint16_t index, const adsb_vehicle_t &vehicle)
         return;
     }
     in_state.vehicle_list[index] = vehicle;
+
+    write_log(vehicle);
```

Comment:
Is this the correct place to be logging this?

Reply:
I think so, this just logs (or not depending on the param) all the ADSB messages received over mavlink